cpp/benchmarks/search/contains.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <benchmarks/common/generate_input.hpp>
+#include <benchmarks/fixture/rmm_pool_raii.hpp>
+
+#include <cudf/detail/search.hpp>
+#include <cudf/scalar/scalar_factories.hpp>
+#include <cudf/types.hpp>
+
+#include <nvbench/nvbench.cuh>
+
+namespace {
+template <typename Type>
+std::unique_ptr<cudf::table> create_table_data(cudf::size_type n_rows,
+                                               cudf::size_type n_cols,
+                                               bool has_nulls = false)
+{
+  data_profile profile;
+  profile.set_cardinality(0);
+  profile.set_null_frequency(has_nulls ? 0.1 : 0.0);
+  profile.set_distribution_params<Type>(
+    cudf::type_to_id<Type>(), distribution_id::UNIFORM, Type{0}, Type{1000});
+
+  auto const seed = static_cast<uint32_t>(time(nullptr));
```

Comment:
is some determinism expected in benchmarks? setting different seed every time will yield different data each time. Anyway there will be noise in benchmark time results for same data. So, I would like to know the general consensus on usage of random seed in benchmarks.

Reply:
If you look at other benchmarks, it seems to me that they all just fix a seed value. I think that should be sufficient.